### PR TITLE
Fixed a bug where the initial color could not be selected.

### LINKED ISSUE #713: Unable to select first color with `ColorPicker` component
### Which package has an issue?

@yamada-ui/react

### Package Version

1.2.4

### Description

When you click on the `ColorPicker` and the popover opens, when you select a color within the popover, the first color selected is not reflected in the `input`.

### Link to Reproduction

https://hirotomoyamada.github.io/yamada-ui/?path=/story/components-forms-colorpicker--basic

### Steps to Reproduce

1. Click on `ColorPicker`
2. Choice value in popover.
3. See input value.

### Browser

Chrome

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux

### Do you know how to fix the issue?

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix?

Yes

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #713

## Description

Fixed a bug where the initial color could not be selected.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/components/color-picker/src/use-color-selector.ts [hunk 1]
```diff
@@ -195,6 +195,7 @@ export const useColorSelector = ({
   const [parsedValue, setParsedValue] = useState<Hsva>(
     convertHsva(value, fallbackValue),
   )
+
```
Comment (umaidashi) [inline packages/components/color-picker/src/use-color-selector.ts:198]: Is this line break a mistake?

File: stories/components/forms/color-picker.stories.tsx [hunk 2]
```diff
@@ -288,6 +288,8 @@ export const customEyeDropperIcon: Story = () => {
 export const customControl: Story = () => {
   const [value, onChange] = useState<string>("#4387f4ff")
 
+  console.log(value)
```
Comment (umaidashi) [inline stories/components/forms/color-picker.stories.tsx:291]: The console remains in place.

**Review by umaidashi — COMMENTED** [review]
You may need a slight modification.

File: packages/components/color-picker/src/use-color-selector.ts (hunk 1 above)
Comment (hirotomoyamada) [reply packages/components/color-picker/src/use-color-selector.ts]: Oh, sorry. removed line.

File: stories/components/forms/color-picker.stories.tsx (hunk 2 above)
Comment (hirotomoyamada) [reply stories/components/forms/color-picker.stories.tsx]: Oh, removed `console.log`.

**Review by umaidashi — APPROVED** [review]
LGTM

**Pull request merged**